test/specs/commands/set.js

```diff
@@ -819,6 +815,112 @@ describe('set command', () => {
 									},
 								);
 							});
+							Given(
+								'a value "ed14889723f24ecc54871d058d98ce91ff2f973192075c0155ba2b7b70ad2500"',
```

Comment:
It's a little confusing having a nethash that is so similar to mainnet. Can we have a totally different one?

Reply:
let's change it to unlikely happening value like `aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa`?